### ISSUE TITLE
Regression reading ASCII tables (with workaround)

### ISSUE THREAD

**Issue by bogdanni** [nom-tam-fits/nom-tam-fits#466]
There has been a regression wrt version 1.17.1 for reading the header of the file at http://swhv.oma.be/test/solo_L2_phi-hrt-icnt_20230407T032009_V02cavity_0344070601.fits.gz

Stack trace:
```
Exception in thread "main" nom.tam.fits.FitsException: Unrecognizable header in dataFactory
	at nom.tam.fits.FitsFactory.dataFactory(FitsFactory.java:315)
	at nom.tam.fits.Fits.readHDU(Fits.java:1008)
	at nom.tam.fits.Fits.readToEnd(Fits.java:1036)
	at nom.tam.fits.Fits.read(Fits.java:948)
```

**Comment by bogdanni** [comment]
Trying to attach the whole file.
[solo_L2_phi-hrt-icnt_20230407T032009_V02cavity_0344070601.fits.gz](https://github.com/nom-tam-fits/nom-tam-fits/files/12174216/solo_L2_phi-hrt-icnt_20230407T032009_V02cavity_0344070601.fits.gz)

**Comment by attipaci** [comment]
OK, I think I see the issue. As a workaround try set `FitsFactory.setUseAsciiTables(true)`. If that works, I will know for sure what the problem is to fix on master (and for the next release), and we'll also have a workaround in the meantime.

**Comment by attipaci** [comment]
Great, I'll fix that on master (SNAPSHOT), and will be included in 1.18.1 (around 9/15)
